packages/actions/src/uploadCoverage/action.yml

```diff
@@ -4,85 +4,99 @@ runs:
   using: 'composite'
```

Comment:
add a required input (supposedly required doesn't do anything when called from a workflow file but whatever) to this workflow called token and pass the token in.